06_neighborhoods/Transportation/transit_trips_county.R

```diff
@@ -0,0 +1,203 @@
+###################################################################
+
+# Transit Metrics: Transit Trips
+# Tina Chelidze 
+# 2023-2024 Mobility Metrics update
+# Process (for overall, not subgroup, data):
+# 1. For County: Import and reshape data
+# 2. QC checks
+# 3. Add Data Quality marker
+# 4. Export final files
+
+###################################################################
+
+
+# Libraries you'll need
+library(tidyr)
+library(dplyr)
+library(readr)
+library(ggplot2)
+library(tidyverse)
+library(purrr)
+
+# SUMMARY-LEVEL VALUES
+# Raw data pulled from https://htaindex.cnt.org/download/
+# the Housing and Transportation (H+T) Affordability Index from the Center for Neighborhood Technology (CNT)
+
+###################################################################
+
+# 1. Import all the files (and/or combine into one file) with only the relevant variables and years
+
+# FOR COUNTIES - import the raw data files
+# Unlike for Places and Tracts, the CNT website allows you to download raw data for all counties at once
+# so we just import the all-county file for the two available years; 2015 and 2019
+# 2015
+transport_county_2015 <- read_csv("C:/Users/USERNAME/Box/Lab/Projects/Gates Upward Mobility Framework/Outreach and Tools/Data/Metrics_2024_round/Transportation/htaindex2015_data_counties.csv")
```

Comment:
Ideally code would download the CSV from the website into a temp folder and then read it in. It looks like the data is behind a login screen so this may not be possible but is worth investigating as it is good practice.

Otherwise the issue with Box Drive is that the path is permission specific. For example I have permission to  the "Data" folder but not any of the root folders. If we use Box Drive I suggest creating a single line of code at the top where the person running the code can put their file path, then use past0 in the read_csvs.